docs/intro.asciidoc

```diff
@@ -72,7 +73,49 @@ https://unpkg.com/@elastic/apm-rum/dist/bundles/elastic-apm-rum.umd.min.js[UNPKG
 </script>
 ----
 
-NOTE: Currently our minified JavaScript bundle is about 16KB (gzipped).
+NOTE: Currently the optimized(minified + gzipped) JavaScript bundle size is about 16KiB.
+
+
+[float]
+[[using-production-build]]
+=== Using Production Build
+
+By default, RUM agent logs all the debug messages to the browser console. These logs are very useful in development. However, they make the RUM agent bundle size larger so you should make sure to use the optimized production version when deploying your application.
```

Comment:
This sentence is a bit confusing to me: "However, they make the RUM agent bundle size larger so you should make sure to use the optimized production version when deploying your application." What about reworking the entire paragraph? Something like this reads a bit easier to me:

```suggestion
By default, the RUM agent logs all debug messages to the browser console.
While these logs are very useful in development, they increase the RUM agent bundle size.
For this reason, make sure to use the optimized production version when deploying your application.
You can find instructions for building optimized code, with different bundlers, below.
```

Reply:
Yeah simpler, As always thanks @bmorelli25 😄